### ISSUE TITLE
Improve the show-instances in `defErrorHandler` and `makeREPLSimple`

### ISSUE THREAD

**Issue by jtapolczai** [ombocomp/repl-toolkit#13]
Currently, the native Show-instance is used to display exceptions, which is quite ugly.

While the Show-instances should be kept, the error message displayed to the user should be improved.

**Comment by jtapolczai** [comment]
Already fixed.

**Comment by jtapolczai** [comment]
The show-instance is still quite ugly, especially in the case of wrapped exceptions, e.g.

``` haskell
> :x
Enter exlusions file: test
AskerPredicateError (GenericPredicateError "test could not be read.")
```

**Comment by jtapolczai** [comment]
Closed by e5015a0c2b3371c81a3f73c1299686f6d26804d8.
